javascript/testing/testing-1.md

```diff
@@ -14,3 +21,12 @@ This lesson is going to center around __Jest__. The biggest reasons for this dec
 4. Read and follow the [Using Matchers](https://facebook.github.io/jest/docs/en/using-matchers.html#content) document on the main Jest website.  This one demonstrates some of the other useful functions you can use in your tests.
 5. Watch [this amazing video](https://www.youtube.com/watch?v=URSWYvyc42M) that covers _what_ to test in your codebase.  The video is specifically about testing the Ruby language, but that doesn't matter _at all_.  The concepts here ring true in any language, and luckily Ruby is a clear enough language that you will be able to follow along just fine.  This video might be worth re-visiting after you've done some testing of your own.
 </div>
+
+### Additional Resources
```

Comment:
I'm not sure if we want this section here with no resources in it, but I'm not sure if we have any official position on this.

I would love input from any other maintainers regarding this.

Reply:
I went ahead and did this because the html/css rewrite content had empty additional resources to encourage people to contribute.

Reply:
👍 confirmed with the team that this is a pattern we've settled on now.